packages/transactions/src/utils.ts

```diff
@@ -206,3 +206,11 @@ export const validateStacksAddress = (stacksAddress: string): boolean => {
     return false;
   }
 };
+
+export const validateTxId = (tx_id: string): boolean => {
```

Comment:
This function has been updated here https://github.com/blockstack/stacks-wallet-web/blob/a6ac94ab12169d55136349991d135f5ea25f25dc/src/common/validation/validate-tx-id.ts for a different/cleaner implementation. Just a suggestion, both work

Reply:
Alright i will update this as well.

Reply:
@aulneau Added these changes.